pandas-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -1,23 +1,37 @@
-"""Project settings."""
-from {{cookiecutter.python_package}}.hooks import ProjectHooks
+"""Project settings. There is no need to edit this file unless you want to change values
+from the Kedro defaults. For further information, including these default values, see
+https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html."""
 
-# Instantiate and list your project hooks here
-HOOKS = (ProjectHooks(),)
+# Instantiated project hooks.
+# from {{ cookiecutter.python_package }}.hooks import ProjectHooks
+# HOOKS = (ProjectHooks(),)
```

Comment:
Irrelevant to the purpose of this PR, but is `ProjectHooks()` even exist now? I don't think it exists maybe we should remove it from the comments too.

Reply:
Not quite sure, @AntonyMilneQB?

Reply:
If a user has custom hooks they would still need to registered here in `settings.py` so that's why we've just left it commented out as a way of showing how to add hooks if needed.